shopify-features/models/profiles.yaml

```diff
@@ -140,26 +184,19 @@ models:
 
         - entity_var:
             name: days_since_first_purchase
-            select: "{% if !(end_time|isnil) %} datediff(day, date(min_timestamp_order_created), date('{{end_time.Format(\"2006-01-02 15:04:05\")}}')){% else %} 1=1 {% endif %}"
-            description: Number of days since the user purchased the first product
-    #has credit card
+            # select: datediff(day, date(min_timestamp_order_created), date('{{timestamp.Format("2006-01-02 15:04:05")}}'))
+            select: "{{macro_datediff('min_timestamp_order_created')}}"
+    # has credit card
         - entity_var:
             name: has_credit_card
             select: max(case when lower(payment_details_credit_card_company) in ('visa','american express','mastercard') then 1 else 0 end)
             from: inputs/rsOrderCreated
             description: If the user has a credit card.
-    #avg_units_per_transaction
-        - entity_var:
-            name: avg_units_per_transaction
-            select: avg({{ array_size ( 'products' )}}::real)
-            from: inputs/rsOrderCreated
-            description: It shows the average units purchased in each transaction. (Total units in each transaction/Total transactions). Includes only those transactions where the total price (from column current_total_price) is greater than zero. So, the feature exclude transactions with 100% off, replacement products etc that may result in the total_price being equal to zero.
-
-    #avg_transaction_value
+    
+             
```

Comment:
The logic changed from here to what we have at the top. Let's use this logic (from current main) and place it here, so the diff is easily visible. Description is missing in the current entity_var above for example.